bin/pycbc_banksim

```diff
@@ -336,12 +369,25 @@ with ctx:
         # If not set fall back on filter low-freq cutoff
         if f_lower < 0.000001:
             f_lower = options.filter_low_frequency_cutoff
+        if f_lower < options.filter_low_frequency_cutoff:
```

Comment:
Not sure if a warning is the best thing here, I feel they tend to get unnoticed or ignored. Why not just raising an error explaining what is going on? Then the user is forced to lower the filter low-frequency cutoff accordingly (and hopefully think twice about what he/she is doing).